Contributing.md

```diff
@@ -21,6 +21,10 @@
   * Run 'npm install' so that the package-lock.json is updated
   * Push changes in a PR
   * Follow PR until it is approved/merged
+  
+# Next steps after release
+
+It is highly recommended to integrate the newly released version of VS Code Camel K in Eclipse Che registry. See [Contributing.md in Che registry](https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/redhat/vscode-camelk/Contributing.md).

```

Comment:
Language can be improved a little here. I would get rid of "it is highly recommended", since all of these are just steps in a process and this should definitely be done if the rest of the steps are done too. 

"Create a PR to integrate the newly released version of VS Code Camel K in the Eclipse Che Registry."

Reply:
it can be several PRs if it requires an update of the sidecar or not.